apps/judicial-system/backend/migrations/20220204105457-update-defendant.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn(
+          'defendant',
+          'no_national_id',
+          {
+            type: Sequelize.STRING,
```

Comment:
Are booleans represented as strings in the backend?

Reply:
absolutely not